get-started/code-experiments/README.md

```diff
@@ -0,0 +1,260 @@
+## DVC Get Started for Experiments
+
+This repository contains a project that trains a classifier for [Fashion
+MNIST][fmnist] dataset in Tensorflow. 
+
+[fmnist]: https://github.com/zalandoresearch/fashion-mnist
+
+It is used as a showcase for [`dvc exp`][gsdvcexp] commands to manage large
+number of experiments. 
+
+[gsdvcexp]: https://dvc.org/doc/start/experiments
+
+After [installing DVC][installdvc] and cloning the repository, you can run:
+
+[installdvc]: https://dvc.org/doc/install
+
+```console
+python -m venv .venv
+. .venv/bin/activate
+python -m pip install -r requirements.txt
+```
+
+`params.yaml` defines many parameters to modify with `dvc exp run --set-param/-S`
+option. For example to set the number of convolutional units in `cnn` model, you
+can use:
+
+```console
+dvc exp run -S model.cnn.conv_units=128 
+```
+
+The experiment will produce a new `metrics.json` and you can check the changes
+in metrics:
+
+```console
+dvc exp diff
+```
+
+You can queue many experiments with `--queue` option:
+
+```console
+dvc exp run --queue -S model.cnn.dropout=0.1
+dvc exp run --queue -S model.cnn.dropout=0.2
+dvc exp run --queue -S model.cnn.dropout=0.3
+dvc exp run --queue -S model.cnn.dropout=0.4
+dvc exp run --queue -S model.cnn.dropout=0.5
+dvc exp run --queue -S model.cnn.dropout=0.6
+dvc exp run --queue -S model.cnn.dropout=0.7
+dvc exp run --queue -S model.cnn.dropout=0.8
+dvc exp run --queue -S model.cnn.dropout=0.9
+```
+
+and run them at once, preferably in parallel with `--jobs`:
+
+```console
+dvc exp run --run-all --jobs 4
+```
+
+You can get the summary of experiments with: 
+
+```console
+dvc exp show
+```
+
+and limit the parameters and metrics to show with `--include-params` and
+`--include-metrics` options, respectively.  
+
+Experiments are given hash-names derived from their inputs and environment. It
+may be easier to review them when you give names with `--name/-n` option.
+
+```console
+dvc exp run -n baseline-experiment
+```
+
+Experiments are normally not checked-in to Git. When there is an experiment that
+you want to preserve in Git history, you can use:
+
+```console
+dvc exp apply exp-123456
+```
+
+Then you can use `git add`, `git commit`, `git push` and `dvc push` as usual. 
+
+You can push and pull the _code changes_ related to an experiment with `dvc exp
+push` and `dvc exp pull` respectively. These two commands work with _Git
+remotes._
```

Comment:
```suggestion
You can push and pull the _code changes_ related to an experiment with
`dvc exp push` and `dvc exp pull` respectively. These two commands work with
_Git remotes._
```

Reply:
`p` of `dvc exp` is on column 79, why do you break? MD converter should be smart enough to convert `dvc exp
push`

Reply:
Yeah I guess it doesn't matter here (README file). It would need to be broken if it was content for the docs engine though (per https://dvc.org/doc/user-guide/contributing/docs).